test/e2e.contract.deploy.js

```diff
@@ -9,11 +9,13 @@ describe('contract.deploy [ @E2E ]', function() {
     var accounts;
     var basic;
     var reverts;
+    var noBytecode;
     var options;
 
     // Error message variants
-    var ganacheRevert = "revert";
-    var gethRevert = "code couldn't be stored";
+    var revertMessage = "revert";
+    var couldNotBeStoredMessage = "code couldn't be stored";
+    var creationWithoutDataMessage = "contract creation without any data provided";
```

Comment:
This message emitted by geth, code ref here 

https://github.com/ethereum/go-ethereum/blob/7a509b47321a28e36f437170a3cc5c283ee7909d/internal/ethapi/api.go#L1404-L1406